src/Api/Controllers/AccountsController.cs

```diff
@@ -194,6 +195,30 @@ public async Task PostPassword([FromBody]PasswordRequestModel model)
             await Task.Delay(2000);
             throw new BadRequestException(ModelState);
         }
+        
+        [HttpPost("password-no-compare")]
+        public async Task PostPasswordNoCompare([FromBody]PasswordNoCompareRequestModel model)
+        {
+            var user = await _userService.GetUserByPrincipalAsync(User);
+            if (user == null)
+            {
+                throw new UnauthorizedAccessException();
+            }
+
+            var result = await _userService.ChangePasswordNoCompareAsync(user, model.NewMasterPasswordHash, model.Key);
+            if (result.Succeeded)
+            {
+                return;
+            }
+
+            foreach (var error in result.Errors)
+            {
+                ModelState.AddModelError(string.Empty, error.Description);
+            }
+
+            await Task.Delay(2000);
```

Comment:
Why is there an explicit delay programmed into here? @kspearrin , I see this in the `PostPassword()` endpoint above as well, is this to prevent brute-force against this endpoint?

Reply:
I am unsure - will defer to Kyle.

Reply:
Yes, this is done to slow down brute force. Not sure it applied here since we are not authenticating from a password.

Reply:
You can probably remove this for this API